docs/source/AdministratorGuide/Systems/MonitoringSystem/index.rst

```diff
@@ -47,18 +48,48 @@ For example::
        User = test
        Password = password
      }
+   }
+
+
+The following option can be set in `Systems/Monitoring/<Setup>/Databases/MonitoringDB`:
+
+   *IndexPrefix*:  Prefix used to prepend to indexes created in the ES instance. If this
+		   is not present in the CS, the indexes are prefixed with the setup name.
 
+For each monitoring types managed, the Period (how often a new index is created)
+can be defined with::
+
+   MonitoringTypes
+   {
+     ComponentMonitoring
+     {
+       # Indexing strategy. Possible values: day, week, month, year, null
+       Period = month
+     }
+     RMSMonitoring
+     {
+       # Indexing strategy. Possible values: day, week, month, year, null
+       Period = month
+     }
+     WMSHistory
+     {
+       # Indexing strategy. Possible values: day, week, month, year, null
+       Period = day
+     }
    }
 
+The given periods above are also the default periods in the code.
+
 
 Enable WMSHistory monitoring
 ============================
 
-You have to install the WorkloadManagemet/StatesMonitoringAgent. This agent is used to collect information using the JobDB and send it to the Elasticsearch database.
-If you install this agent, you can stop the StatesAccounting agent.
+You have to install the :mod:`~DIRAC.WorkloadManagementSystem.Agent.StatesMonitoringAgent`.
```

Comment:
So this isn't true anymore, isn't it? One doesn't have to install this agent, instead one can install the StatesAccountingAgent and set Backends=Monitoring